source/org/zfin/profile/presentation/PersonController.java

```diff
@@ -92,6 +92,9 @@ public String editView(@PathVariable String zdbID, Model model) {
         }
         model.addAttribute("showDeceasedCheckBox", showDeceasedCheckBox);
         model.addAttribute("countryList", profileService.getCountries());
+
+        model.addAttribute("emailPrivacyPreferenceList", profileRepository.getAllEmailPrivacyPreferences());
```

Comment:
Maybe, this duplicated code could also be put into one class and be re-used.

Reply:
I updated it to use the profileService